app/controllers/broadcasts_controller.rb

```diff
@@ -41,17 +42,14 @@ def new_with_users
 
     @broadcast = Broadcast.accessible_by(current_ability).new
     users = User.accessible_by(current_ability).with_assoc.by_name
+    users = apply_search_if_given(Question, users)
```

Comment:
`Question`? Should this be `User`? Can you please test this manually or add a spec?

Reply:
fixed, i'm going to make a truth table for broadcasts specs as you said later